homeassistant/components/media_player/emby.py

```diff
@@ -234,18 +251,37 @@ def media_duration(self):
                 return None
 
     @property
+    def media_position(self):
+        """Position of current playing media in seconds."""
+        if self.media_duration is None:
+            return None
+
+        try:
+            return int(self.session['PlayState']['PositionTicks']) / 10000000
```

Comment:
Good point.  I added the update logic after defining these properties and forgot to loop back to simplify.